src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -420,6 +444,34 @@ private static void close(final Closeable res) {
             this.output = out;
             this.level = lvl;
         }
+
+        /**
+         * Checks if line is part of a stack trace and should be appended.
```

Comment:
@dean-e-clark does it mean that stactraces can have just these 3 prefixes and otherwise they are not stacktraces?